tests/end_to_end/end_to_end.cpp

```diff
@@ -659,3 +662,64 @@ TEST_CASE("enumerate_and_query_programs", "[enumerate_and_query_programs]")
     REQUIRE(ebpf_api_get_next_program(program_handle, &program_handle) == EBPF_SUCCESS);
     REQUIRE(program_handle == INVALID_HANDLE_VALUE);
 }
+
+TEST_CASE("pinned_map_enum", "[pinned_map_enum]")
+{
+    ebpf_handle_t map_handle;
+    uint32_t result = 0;
+    const int pinned_map_count = 10;
+    std::string pin_path_prefix = "\\ebpf\\map\\";
+    uint16_t map_count = 0;
+    ebpf_map_information_t* map_info = NULL;
```

Comment:
nullptr

Reply:
Fixed. Code moved to tests/libs/common/common_tests.cpp